### PR TITLE
Adjusted Brightness for Show QR code

### LINKED ISSUE #25: No change in brightness on viewing QR Code
**Summary:**

There is no change in brightness of device when QR Code is shown to the user.

**Steps to reproduce:**

Transfer -> Show Code

**Expected behavior:**

The brightness of device can be maximised when clicked on "Show Code" and returned to normal when pressed back.

Can I work on this? @naman14

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixes #25 

Please Add Summary of what changes you have made.(Optional)
Adjusted Brightness to FULL for Show QR code Activity

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by luckyman20** [comment]
@codepoet2017390 Please upload gif showing your changes.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java [hunk 1]
```diff
@@ -52,6 +53,10 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         mShowQrPresenter.generateQr(qrData);
         tvQrData.setText(qrData);
 
+        WindowManager.LayoutParams layout = getWindow().getAttributes();
+        layout.screenBrightness = 1F;
+        getWindow().setAttributes(layout);
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java]:
```suggestion
```

**Comment by luckyman20** [comment]
@codepoet2017390 Push your changes to `dev` branch.

**Comment by codepoet2017390** [comment]
@luckyman20 i think deleting that space is causing travis to fail

**Pull request closed without merging**